doc/changes/changes-4.10.xml

```diff
@@ -0,0 +1,312 @@
+<Chapter Label="ChangesGAP49toGAP410"> 
+<Heading>Changes between &GAP; 4.9 and &GAP; 4.10</Heading>
+
+This chapter contains an overview of the most important changes
+introduced in &GAP; 4.10.0 release (the 1st public release of &GAP; 4.10).
+Later it will also contain information about subsequent update 
+releases for &GAP; 4.10.
+<P/>
+
+These changes are also listed on the Wiki page
+<URL>https://github.com/gap-system/GAP/wiki/gap-4.10-release-notes</URL>.
+
+<Section Label="GAP4.10.1">
+<Heading>&GAP; 4.10.0 (November 2018)</Heading>
+
+<Subsection> 
+<Heading>New features and major changes</Heading>
+
+<List>
+
+<!-- 
+ -->
+<Mark>Reduce impact of immediate methods</Mark><Item>
+
+GAP allows declaring so-called "immediate methods". The idea is that these
+are very simple and fast methods which are immediately called if
+information about an object becomes known, in order to perform some quick
+deduction. For example, if the size of a group is set, there might be
+immediate methods which update the filters <C>IsFinite</C> and <C>IsTrivial</C> of the
```

Comment:
Critical: while converting all markdown to gapdoc, one should convert all `Things` to `Ref` elements for cross-references to appropriate manual sections. This should definitely be done by someone, possibly me, before release.

Reply:
In an ideal world, this should be done. But that is a ton of work for little to no reward, so I won't do it, and I'd suggest you don't waste your time for it either. I know no other project which does something similar, or would even consider spending that much time on release note.

In particular, not all `<C>...</C>` mark references. And it is lot of work to properly classify them as Attr/Prop/Func/etc.

Reply:
> In particular, not all <C>...</C> mark references. And it is lot of work to properly classify them as Attr/Prop/Func/etc.

Exactly. That part - figuring out what may be `<Ref>` and what not, and also the type (which does have a correctness check) took a lot of time previously. But If I will decide to abandon this approach, at least some important things should be cross-referenced...

Reply:
P.S. I think that useful for cross-referencing are things like new functionality, changed functionality etc. - when there may be a real need for the user to follow the link. Not for the `filters <C>IsFinite</C> and <C>IsTrivial</C>` above, but e.g. when introducing new `command line options <C>--norepl</C> and <C>--nointeract</C>`. Of course, that can also be done while polishing the manual later, but in practice one would rarely or never return to overviews of past releases.